.travis.yml

```diff
@@ -57,5 +57,6 @@ addons:
     postgresql: "9.3"
 
 script:
+    - make clean_travis
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then make travis; fi
     - make test
```

Comment:
I think we could also add a make clean at the end just to make sure that none of the scripts/command exit unexpectedly.